package.json

```diff
@@ -1,18 +1,31 @@
 {
```

Comment:
For production code `^` need to be deleted to lock in the version for some libraries, for the purpose of this test, i left it as it si